core/src/main/scala/stryker4s/mutants/findmutants/FileCollector.scala

```diff
@@ -1,29 +1,102 @@
+    */
   override def collectFilesToMutate(): Iterable[File] = {
-    filesToMutate.filterNot(file => filesToExcludeFromMutation.contains(file))
+    filesToMutate
+      .filterNot(filesToExcludeFromMutation.contains(_))
+      .filterNot(stryker4sTmpFiles.contains(_))
   }
 
-  private[this] val filesToMutate: Seq[File] = {
-    config.mutate
-      .filterNot(file => file.startsWith("!"))
+  /**
+    * Collect all files that are needed to be copied over to the Stryker4s-tmp folder.
+    *
+    * Option 1: Copy every file that is listed by the 'files' config setting
+    * Option 2: Copy every file that is listed by git.
+    * Option 3: Copy every file in the 'baseDir' excluding target folders.
+    */
+  override def filesToCopy(processRunner: ProcessRunner): Iterable[File] = {
+    (listFilesBasedOnConfiguration() orElse listFilesBasedOnGit(processRunner) orElse {
+      warn("No 'files' specified and not a git repository.")
+      warn("Falling back to copying everything except the target/ folder(s)")
+
+      listAllFiles()
+    }).getOrElse(Seq.empty)
+  }
+
+  /**
+    * List all files based on the 'files' configuration key from stryker4s.conf.
+    */
+  private[this] def listFilesBasedOnConfiguration(): Option[Iterable[File]] = {
+    debug("Collecting files based on 'files' configuration key.")
+    config.files.map { glob }
+  }
+
+  /**
+    * List all files based on `git ls-files` command.
+    */
+  private[this] def listFilesBasedOnGit(processRunner: ProcessRunner): Option[Iterable[File]] = {
+    debug("Collecting files based on 'git ls-files'.")
+    processRunner(Command("git ls-files", "--others --exclude-standard --cached"), config.baseDir) match {
+      case Success(files) => Option(files.map(config.baseDir / _).distinct)
+      case Failure(_)     => None
+    }
+  }
+
+  /**
+    * List all files from the base directory specified in the Stryker4s basedir config key.
+    */
+  private[this] def listAllFiles(): Option[Iterable[File]] = {
+    Option(config.baseDir
+      .listRecursively
+      .filterNot(file => file.pathAsString.contains(s"${pathSeparator}target$pathSeparator"))
+      .toIterable)
+  }
+
+  private[this] def glob(list: Seq[String]): Seq[File] = {
+    list
       .flatMap(config.baseDir.glob(_))
       .distinct
   }
 
+  private[this] val filesToMutate: Seq[File] = {
+    glob(config.mutate.filterNot(file => file.startsWith("!")))
+  }
+
   private[this] val filesToExcludeFromMutation: Seq[File] = {
-    config.mutate
-      .filter(file => file.startsWith("!"))
-      .flatMap(file => config.baseDir.glob(file.stripPrefix("!")))
-      .distinct
+    glob(
+      config.mutate
+        .filter(file => file.startsWith("!"))
+        .map(file => file.stripPrefix("!")))
+  }
+
+  /**
+    * List of all previously copied files if the target folder is not cleaned.
+    */
+  private[this] val stryker4sTmpFiles: Seq[File] = {
+    config.baseDir
+      .glob(s"target${pathSeparator}stryker4s-*")
```

Comment:
I think the `better.files` glob doesn't work with windows path separators, but always uses `/`, so this wouldn't work on windows